### PR TITLE
add account, edit account, delete account, change account status by a…

### PR DESCRIPTION
**Pull request by at-tuannd**
…jax, add colums description in table users

### CODE REVIEW

File: app/Http/Controllers/Admin/UserController.php [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    public function postAdd(UserAddRequest $request)
+    {
+         $input=[
+             'fullname'=>$request->input('fullname'),
+             'email'=>$request->input('email'),
+             'password'=>bcrypt($request->input('password')),
+             'company_name'=>$request->input('companyname'),
+             'company_address'=>$request->input('companyaddress'),
+             'phone'=>$request->input('phone'),
+             'level'=>2,
```
Comment (at-vietpb) [inline app/Http/Controllers/Admin/UserController.php]: What is `2`? Please define this value in config.

File: app/Http/Controllers/Admin/UserController.php [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+             'status'=>1,
+             'descriptions'=>$request->input('descriptions'),
+             'remember_token'=>$request->input('_token')
```
Comment (at-vietpb) [inline app/Http/Controllers/Admin/UserController.php]: You don't need store `remember_token` when create or update the user.

File: app/Http/Controllers/Admin/UserController.php [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+     /**
+     * Delete user.
+     *
+     * @param id $id [descriptions]
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
+    {
+        if (Auth::user()->level==1 || Auth::user()->level!=1 && Auth::user() != $id) {
```
Comment (at-vietpb) [inline app/Http/Controllers/Admin/UserController.php]: Are you sure `Auth::user()` will return `id`?

File: resources/lang/en/messages.php [hunk 4]
```diff
@@ -0,0 +1,11 @@
+<?php
+return [
+'delete_error'=>'Delete errors !',
```
Comment (at-vietpb) [inline resources/lang/en/messages.php]: Please indent lines in the array.

File: public/admin/js/change_status_user.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+		if(status==true){
+			status=1;
+		}else{
+			status=2;
+		}
+		var name=$(this).attr('data-name');
+		var id_user=$(this).attr('data-id');
+		var _token=$('input[name="_token"]').val();
+		$.ajax({
+			url:'/admin/user/change-status',
```
Comment (at-vietpb) [inline public/admin/js/change_status_user.js]: This is a static URL, is not good when URL changes. Please define this URL from blade file.
Example:
**index.blade.php**
```
<script>
    var urlChangeStatus = "{{ route('user.change_status') }}";
</script>
```

**change_status_user.js**
```
url: urlChangeStatus,
```

For the best, you can make a js object to store URLs for js functions.

File: public/admin/js/change_status_user.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+				if(data=='ok'){
+					if(state==true){
+						setTimeout(function(){ alert(name+ ' is active'); }, 500);
+					}else{
+						setTimeout(function(){ alert(name+' is not active'); }, 500);
+					}
+				}
+			},
+			error:function(){
+				alert('lỗi');
```
Comment (at-vietpb) [inline public/admin/js/change_status_user.js]: To log error, you can use `console.log()`

File: resources/views/admin/include/slidebar_menu.blade.php [hunk 7]
```diff
@@ -45,7 +45,7 @@
         </a>
         <ul class="sub-menu">
             <li class="nav-item">
-                <a href="{{ url('khoa-luan-admin/khach-san/dia-diem-khach-san/') }}" class="nav-link ">
+                <a href="{{ url('admin/user/list') }}" class="nav-link ">
```
Comment (at-vietpb) [inline resources/views/admin/include/slidebar_menu.blade.php]: Use `route()` function to get URL from route

File: app/Http/Controllers/Admin/UserController.php [hunk 8]
```diff
@@ -75,10 +75,10 @@ public function postAdd(UserAddRequest $request)
              'company_name'=>$request->input('companyname'),
              'company_address'=>$request->input('companyaddress'),
              'phone'=>$request->input('phone'),
-             'level'=>2,
-             'status'=>1,
+             'level'=>trans('user.level_member'),
```
Comment (at-vietpb) [inline app/Http/Controllers/Admin/UserController.php]: Don't define config value in lang file. Please use config files and `config()` function.

File: resources/views/admin/include/slidebar_menu.blade.php (hunk 7 above)
Comment (at-tuannd) [inline resources/views/admin/include/slidebar_menu.blade.php]: Yes. I will milk after, thank you

**Pull request merged**